pages/storage-page.ts

```diff
@@ -0,0 +1,16 @@
+// Select LVM as the storage device
+
+export class StoragePage {
+  readonly page: Page;
```

Comment:
missing the rest of class property as readonly, ie: useLVMToggle (check playwright example from documentation)